src/main/java/com/twitter/kinesis/utils/Environment.java

```diff
@@ -36,70 +36,69 @@ public void logProperties() {
   }
```

Comment:
we might want to add the replay variables to be loaded from envvars to this ticket too. The replay variables to be loaded are in this pull request https://github.com/offerpop/gnip-kinesis-connector/pull/2. The changes in this pull request need to be added to the sample-kinesis-connector when working on the ticket to add replay support.